packages/mobile-top-nav/src/assets/img/icon_audio.js

```diff
@@ -0,0 +1,55 @@
+import { html, LitElement } from 'lit-element';
+import Icon from './icon';
+
+class IconAudio extends Icon {
+  render() {
+    return html`
+      <svg
+        width="20px"
+        height="24px"
+        viewBox="0 0 20 24"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+      >
+        <g
+          id="Mockups-&amp;-Interactions-(Revised)"
```

Comment:
Suggestion: remove IDs from nested svg elements and defer to using a `<title>` tag at the very least